crates/forge/src/fuzzer.rs

```diff
@@ -0,0 +1,223 @@
+use cairo_felt::Felt252;
+use num_bigint::{BigUint, RandBigInt};
+use num_traits::{One, Zero};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+use std::default::Default;
+
+pub struct RunParams {
+    /// Inclusive value
+    low: BigUint,
+    /// Exclusive value
+    high: BigUint,
+    /// Number of arguments
+    arguments_number: usize,
+    /// Total number of runs
+    total_runs: u32,
+    /// Number of already executed runs
+    executed_runs: u32,
+    /// Run in which an argument has a min value
+    /// e.g. `run_with_min_value_argument[0] = 5`
+    /// means that the first argument will have the lowest possible value in 5th run
+    run_with_min_value_for_argument: Vec<u32>,
+    /// Run in which argument has a max value
+    /// e.g. `run_with_max_value_for_argument[0] = 5`
+    /// means that the first argument will have the highest possible value in 5th run
+    run_with_max_value_for_argument: Vec<u32>,
+}
+
+impl Default for RunParams {
```

Comment:
Do we use it anywhere?

Reply:
Only in unit tests, but it makes the code a little bit shorter.

Reply:
Then I think this code should be in `mod tests`